keycloak/keycloak-admin-ui
# Realm Roles: associated roles modal functionality

## Motivation

Will close https://issues.redhat.com/browse/APPDUX-791

Also addresses https://issues.redhat.com/browse/APPDUX-728

## Brief Description


See #284, #316 

Extends modal with functionality to POST/FETCH associated roles and display them in the realm roles section.

## Verification Steps



1. Go to  Realm Roles section.
2. Create a new role or click on an existing role.
3. Click on the Action dropdown -> select "Add associated roles".
4. Select roles and click "Add".
5. Verify that the "Associated roles" tab is visible.

## Checklist:


## Additional Notes
